app/controllers/votes_controller.rb

```diff
@@ -1,8 +1,15 @@
 class VotesController < ApplicationController
-  def create
-    topic = Topic.find(params[:topic_id])
-    vote = topic.votes.build
-    vote.save!
-    redirect_to(topics_path)
-  end
+  # def create
+  #   topic = Topic.find(params[:topic_id])
+  #   vote = topic.votes.build
+  #   vote.save!
+  #   redirect_to(topics_path)
+  # end
+
+  # def update
```

Comment:
Also if it gets too weird to implement the `destroy` action, create a custom `destroy_last` or `downvote` method, but leave it in this controller instead.

In general, try to have the `VotesController` create `Vote` objects, the `TopicsController` create `Topic` objects, etc. if you can help it.